src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+  }
+  &--1 &__star:nth-child(-n+1),
+  &--2 &__star:nth-child(-n+2),
+  &--3 &__star:nth-child(-n+3),
+  &--4 &__star:nth-child(-n+4),
+  &--5 &__star:nth-child(-n+5) {
+    background-image: url(../images/star-active.svg);
+  }
+  & &__star:nth-child(5) {
+    background-image: url(../images/star.svg);
+  }
```

Comment:
seem almost identical with Ala implementation - so same comment - use a loop according to a checklist

Reply:
fixed